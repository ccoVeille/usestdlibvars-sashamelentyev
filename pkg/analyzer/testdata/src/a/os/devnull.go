// Code generated by usestdlibvars, DO NOT EDIT.

package os_test

import "fmt"

var (
	_ = "/dev/null" // want `"/dev/null" can be replaced by /dev/null`
)

const (
	_ = "/dev/null" // want `"/dev/null" can be replaced by /dev/null`
)

func _() {
	_ = func(s string) string { return s }("/dev/null") // want `"/dev/null" can be replaced by /dev/null`
	_ = func(s string) string { return s }("text before key /dev/null")
	_ = func(s string) string { return s }("/dev/null text after key")
}

func _() {
	_ = fmt.Sprint("/dev/null") // want `"/dev/null" can be replaced by /dev/null`
	_ = fmt.Sprint("text before key /dev/null")
	_ = fmt.Sprint("/dev/null text after key")
}
