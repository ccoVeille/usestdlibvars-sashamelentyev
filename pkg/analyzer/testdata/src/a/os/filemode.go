// Code generated by usestdlibvars, DO NOT EDIT.

package os_test

import "os"

// var f *os.File

var (
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o400) // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_    = os.WriteFile("file.txt", nil, 0o400)        // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_    = os.Chmod("file.txt", 0o400)                 // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_    = os.Mkdir("folder", 0o400)                   // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_    = os.MkdirAll("folder", 0o400)                // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o440) // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_    = os.WriteFile("file.txt", nil, 0o440)        // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_    = os.Chmod("file.txt", 0o440)                 // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_    = os.Mkdir("folder", 0o440)                   // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_    = os.MkdirAll("folder", 0o440)                // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o444) // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.WriteFile("file.txt", nil, 0o444)        // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.Chmod("file.txt", 0o444)                 // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.Mkdir("folder", 0o444)                   // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.MkdirAll("folder", 0o444)                // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o600) // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_    = os.WriteFile("file.txt", nil, 0o600)        // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_    = os.Chmod("file.txt", 0o600)                 // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_    = os.Mkdir("folder", 0o600)                   // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_    = os.MkdirAll("folder", 0o600)                // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o644) // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.WriteFile("file.txt", nil, 0o644)        // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.Chmod("file.txt", 0o644)                 // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.Mkdir("folder", 0o644)                   // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_    = os.MkdirAll("folder", 0o644)                // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o666) // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_    = os.WriteFile("file.txt", nil, 0o666)        // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_    = os.Chmod("file.txt", 0o666)                 // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_    = os.Mkdir("folder", 0o666)                   // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_    = os.MkdirAll("folder", 0o666)                // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o700) // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_    = os.WriteFile("file.txt", nil, 0o700)        // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_    = os.Chmod("file.txt", 0o700)                 // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_    = os.Mkdir("folder", 0o700)                   // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_    = os.MkdirAll("folder", 0o700)                // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o755) // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.WriteFile("file.txt", nil, 0o755)        // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.Chmod("file.txt", 0o755)                 // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.Mkdir("folder", 0o755)                   // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.MkdirAll("folder", 0o755)                // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o775) // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.WriteFile("file.txt", nil, 0o775)        // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.Chmod("file.txt", 0o775)                 // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.Mkdir("folder", 0o775)                   // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_    = os.MkdirAll("folder", 0o775)                // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, 0o777) // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_    = os.WriteFile("file.txt", nil, 0o777)        // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_    = os.Chmod("file.txt", 0o777)                 // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_    = os.Mkdir("folder", 0o777)                   // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_    = os.MkdirAll("folder", 0o777)                // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
)
