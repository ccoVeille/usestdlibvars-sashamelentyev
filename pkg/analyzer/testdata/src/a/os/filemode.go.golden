// Code generated by usestdlibvars, DO NOT EDIT.

package os_test

import "os"

// var f *os.File

var (
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRUSR) // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRUSR) // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_ = os.Chmod("file.txt", syscall.S_IRUSR) // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_ = os.Mkdir("folder", syscall.S_IRUSR) // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_ = os.MkdirAll("folder", syscall.S_IRUSR) // want `"0o400" can be replaced by syscall\.S_IRUSR`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRUSR | syscall.S_IRGRP) // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRUSR | syscall.S_IRGRP) // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_ = os.Chmod("file.txt", syscall.S_IRUSR | syscall.S_IRGRP) // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_ = os.Mkdir("folder", syscall.S_IRUSR | syscall.S_IRGRP) // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_ = os.MkdirAll("folder", syscall.S_IRUSR | syscall.S_IRGRP) // want `"0o440" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.Chmod("file.txt", syscall.S_IRUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.Mkdir("folder", syscall.S_IRUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.MkdirAll("folder", syscall.S_IRUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o444" can be replaced by syscall\.S_IRUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRUSR | syscall.S_IWUSR) // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRUSR | syscall.S_IWUSR) // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_ = os.Chmod("file.txt", syscall.S_IRUSR | syscall.S_IWUSR) // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_ = os.Mkdir("folder", syscall.S_IRUSR | syscall.S_IWUSR) // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_ = os.MkdirAll("folder", syscall.S_IRUSR | syscall.S_IWUSR) // want `"0o600" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.Chmod("file.txt", syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.Mkdir("folder", syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_ = os.MkdirAll("folder", syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IROTH) // want `"0o644" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IROTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IWGRP | syscall.S_IROTH | syscall.S_IWOTH) // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IWGRP | syscall.S_IROTH | syscall.S_IWOTH) // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_ = os.Chmod("file.txt", syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IWGRP | syscall.S_IROTH | syscall.S_IWOTH) // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_ = os.Mkdir("folder", syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IWGRP | syscall.S_IROTH | syscall.S_IWOTH) // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_ = os.MkdirAll("folder", syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IWGRP | syscall.S_IROTH | syscall.S_IWOTH) // want `"0o666" can be replaced by syscall\.S_IRUSR \| syscall\.S_IWUSR \| syscall\.S_IRGRP \| syscall\.S_IWGRP \| syscall\.S_IROTH \| syscall\.S_IWOTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRWXU) // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRWXU) // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_ = os.Chmod("file.txt", syscall.S_IRWXU) // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_ = os.Mkdir("folder", syscall.S_IRWXU) // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_ = os.MkdirAll("folder", syscall.S_IRWXU) // want `"0o700" can be replaced by syscall\.S_IRWXU`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRWXU | syscall.S_IRGRP | syscall.S_IXGRP | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRWXU | syscall.S_IRGRP | syscall.S_IXGRP | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.Chmod("file.txt", syscall.S_IRWXU | syscall.S_IRGRP | syscall.S_IXGRP | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.Mkdir("folder", syscall.S_IRWXU | syscall.S_IRGRP | syscall.S_IXGRP | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.MkdirAll("folder", syscall.S_IRWXU | syscall.S_IRGRP | syscall.S_IXGRP | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o755" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRGRP \| syscall\.S_IXGRP \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.Chmod("file.txt", syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.Mkdir("folder", syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_ = os.MkdirAll("folder", syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IROTH | syscall.S_IXOTH) // want `"0o775" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IROTH \| syscall\.S_IXOTH`
	_, _ = os.OpenFile("file.txt", os.O_CREATE, syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IRWXO) // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_ = os.WriteFile("file.txt", nil, syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IRWXO) // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_ = os.Chmod("file.txt", syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IRWXO) // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_ = os.Mkdir("folder", syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IRWXO) // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
	_ = os.MkdirAll("folder", syscall.S_IRWXU | syscall.S_IRWXG | syscall.S_IRWXO) // want `"0o777" can be replaced by syscall\.S_IRWXU \| syscall\.S_IRWXG \| syscall\.S_IRWXO`
)

